keanu-project/src/main/java/io/improbable/keanu/util/DescriptionUtils.java

```diff
@@ -0,0 +1,61 @@
+package io.improbable.keanu.util;
+
+import io.improbable.keanu.tensor.Tensor;
+import io.improbable.keanu.vertices.Vertex;
+import io.improbable.keanu.vertices.bool.BooleanVertex;
+import io.improbable.keanu.vertices.bool.nonprobabilistic.operators.binary.BooleanBinaryOpVertex;
+import io.improbable.keanu.vertices.intgr.probabilistic.BinomialVertex;
+
+import static io.improbable.keanu.util.DescriptionCreator.createDescriptionAllowingLabels;
+
+class DescriptionUtils {
+
+    static String createIfStringDescription(BooleanVertex predicate, Vertex thn, Vertex els, boolean includeBrackets) {
```

Comment:
I'm not sure what qualifies a function to move out of DescriptionCreator and in to this file - feels a tiny bit arbitrary?